src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1277,7 +1277,10 @@ private AuthenticationResult acquireTokenLocalCall(final CallbackHandler callbac
     }
 
     private boolean promptUser(PromptBehavior prompt) {
-        return prompt == PromptBehavior.Always || prompt == PromptBehavior.REFRESH_SESSION;
+        return prompt == PromptBehavior.Always 
+                || prompt == PromptBehavior.REFRESH_SESSION 
+                // Force broker with PRT support to prompt user. 
```

Comment:
Move the comment to the next line, after the code "|| PromptBehavior.FORCE_PROMPT == prompt;"